lib/ES/Book.pm

```diff
@@ -209,12 +209,18 @@ sub build {
         }
     }
     $pm->wait_all_children();
-    $self->_copy_branch_to_current( $self->current ) if $rebuilding_current_branch;
-    $self->remove_old_branches;
+    $self->_copy_branch_to_current if $rebuilding_current_branch;
+    $update_version_toc |= $self->_remove_old_branches;
     if ( $self->is_multi_version ) {
-        if ( $rebuilding_any_branch ) {
-            printf(" - %40.40s: Writing versions TOC\n", $self->title);
```

Comment:
I don't think the prints here add that much value because this process is pretty quick.